UnityProject/Assets/Scripts/Items/Kitchen/Cookable.cs

```diff
@@ -60,4 +63,12 @@ private void OnBurnUpServer(DestructionInfo info)
 			Spawn.ServerPrefab(CookedProduct, gameObject.RegisterTile().WorldPosition, transform.parent);
 		}
 	}
+
+	[Flags]
+	public enum CookSource
+	{
+		BurnUp = 1,
```

Comment:
Use the shifting method to assign order, I’m on my phone right now but I recall it lookin like this
```
None = 0,
Something = 1 << 1,
Other = 1 << 2,
Another = 1 << 3,
Final = 1 << 4
```
All flag is created automagically.